docs/tutorials/dashboard.md

```diff
@@ -5,6 +5,12 @@ Prowler allows you to run your own local dashboards using the csv outputs provid
 prowler dashboard
 ```
 
+To run Prowler local dashboard with docker, use: 
+
+```sh
+docker run -e dashboard=dashboard <prowler-image>
```

Comment:
What is the purpose of `-e dashboard=dashboard`?

Reply:
Thanks for the catch! I´ve updated it